src/worker.js

```diff
@@ -68,32 +62,28 @@ class Worker {
   msTimeout: number;
   queue: String;
   retryDelay: number;
+  maxRetryDelay: number
   task: Function;
   tid: String;
 
   constructor (opts: Object) {
     defaults(opts, {
       // default non-required user options
       errorCat: ErrorCat,
-      runNow: true,
       // other options
       attempt: 0,
       finalRetryFn: () => { return Promise.resolve() },
-      maxNumRetries: process.env.WORKER_MAX_NUM_RETRIES || 0,
-      msTimeout: process.env.WORKER_TIMEOUT || 0,
-      retryDelay: process.env.WORKER_MIN_RETRY_DELAY || 1
+      maxNumRetries: parseInt(process.env.WORKER_MAX_NUM_RETRIES, 10) || Number.MAX_SAFE_INTEGER,
```

Comment:
Infinity would be less confusing and still work as expected

Ps. I was just bored browsing github. You can ignore my comments if you like lol